lib/utils.pm

```diff
@@ -643,6 +641,7 @@ sub power_action {
     if (check_var('BACKEND', 'svirt')) {
```

Comment:
wouldn't be better if this `IF` condition was moved into the `prepare_system_shutdown` function?

Reply:
Moved.